app/components/GoogleSigninButton.tsx

```diff
@@ -0,0 +1,17 @@
+"use client";
+import styles from "./components.module.css"
+import React from "react";
+import { signIn, signOut, useSession } from "next-auth/react";
+const GoogleSigninButton = () => {
+  return (
+    <button
+      onClick={() => signIn()}
```

Comment:
```suggestion
      onClick={signIn}
```

https://react.dev/learn/responding-to-events#adding-event-handlers

![image](https://github.com/mamao000/intern-training/assets/108644367/757b279f-ea2a-4b2a-9695-97e58cf6aee2)